normandy/recipes/tests/test_api.py

```diff
@@ -228,6 +229,17 @@ def test_it_serves_recipes(self, api_client):
         assert res.status_code == 200
         assert res.data[0]['name'] == recipe.name
 
+    def test_it_can_create_recipes(self, api_client):
+        action = ActionFactory()
+
+        res = api_client.post('/api/v1/recipe/', {'name': 'Test Recipe',
+                                                  'action_id': action.id,
+                                                  'arguments': '{}'})
```

Comment:
I think you want this to be `'arguments': {}`. That makes the arguments empty object (which is valid), whereas `'{}'` makes the arguments a string containing two brackets, which isn't valid arguments for an action. We should probably actually be enforcing that somehow.